<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefConfigBase</class>
 <widget class="QWidget" name="PrefConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chkOverride">
     <property name="text">
      <string>&amp;Override global settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWnd">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QWidget" name="addWnd" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
